tfjs-backend-webgl/src/conv_backprop_packed_gpu.ts

```diff
@@ -0,0 +1,101 @@
+/**
+ * @license
+ * Copyright 2023 Google LLC. All Rights Reserved.
```

Comment:
nit: I remember @mattsoulanille ever mentioned `All Rights Reserved.` is not needed anymore for new generated file.

Reply:
Nice catch, thanks!